airflow/operators/trigger_dagrun.py

```diff
@@ -156,6 +166,11 @@ def execute(self, context: Dict):
             else:
                 raise e
 
+        # Store the execution date from the dag run (either created or found above) to
+        # be used when creating the extra link on the webserver.
+        ti = context['task_instance']
+        ti.xcom_push(key=XCOM_EXECUTION_DATE_ISO, value=dag_run.execution_date.isoformat())
```

Comment:
```suggestion
        ti.xcom_push(key=XCOM_EXECUTION_DATE_ISO, value=dag_run.execution_date.isoformat())
        ti.xcom_push(key="run_id", value=dag_run.run_id)
```

Too plese

Reply:
Hey @ashb, Sure I can make this change, but I'm curious, why do you want the run id pushed as well? Is there some other feature that depends on it? It isn't required for this feature as far as I can tell.

Reply:
Not _right_ now, but I am slowly working towards making Airflow be able to have multiple runs for the _same_ date, so by adding this now it future proofs us and means we can change the URL without having to worry about backwards compat of "did we push run_id or not"

Reply:
Thanks for the context, it's helpful to add along with requested code changes! I'll push an update to the PR shortly.